src/backend/src/main/java/com/becon/opencelium/backend/execution/support_file/SupportFileServiceImp.java

```diff
@@ -215,9 +210,8 @@ public void collectFiles(Long connectionId, long executionId, String timestamp,
             ConnectionDTO dto = connectionSqlService.getFullConnection(connectionId);
 
             // Add Connection resource template as a JSON file:
-            ConnectionOldDTO oldDTO = dto2OldDtoMapper.toDTO(dto);
-            CtionTemplateResource template = oldDto2ResourceMapper.toDTO(oldDTO);
-            connectionTitle = template.getTitle();
+            TemplateResource template = templateService.getByConnectionId(connectionId);
+            connectionTitle = template.getName(); // Connection.title == Template.name
```

Comment:
`TemplateResource.name` is set to `CtionTemplateResource.title` in [TemplateServiceImp.getByConnectionId](https://github.com/opencelium/opencelium/blob/7fe3660747b67f41faf56ebe2fea09ea7000a5a8/src/backend/src/main/java/com/becon/opencelium/backend/template/service/TemplateServiceImp.java#L160), so we just need to replace `getTitle()` to `getName()`